static-assets/components/cstudio-forms/controls/numeric-input.js

```diff
@@ -106,11 +106,11 @@ YAHOO.extend(CStudioForms.Controls.numericInput, CStudioForms.CStudioFormField,
 	 * @param el element
 	 */
 	count: function(evt, countEl, el) {
-		var max = this.getAttribute("max") ,
-			min = this.getAttribute("min");
-		if(max && parseInt(this.value) > max){
+		var max = el.maxValue,
```

Comment:
Where do these values come from and why is it a good idea to replace the former getattr call?

Reply:
Those values (maxValue/minValue) belong to the element, from defined supportedProperties. The `getAttribute` calls were failing since they're not defined (maybe a copy/paste from somewhere else?).